pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
@saurabh500 Does that mean the managed code in SqlClient library will fail when running on the 1.0.0 version of SNI? As you probably have noticed we are trying to eliminate our TFS dependencies which is why we are trying to only ship the changes in corefx/coreclr repo's, if you guys are still making active changes in TFS we are going to need to work with you to move that code out of TFS. 

@gkhanna79 I do admin that SNI is one library I know little about and don't follow it closely and so I wasn't aware it had interesting changes. We will need to decide if there are enough changes to warrant the cost of another TFS branch.